docs/readmes/nms/alert.md

```diff
@@ -0,0 +1,322 @@
+|authentication_failure	|Auth failure	|AGW	|No	|MME	|
+|nas_auth_rsp_timer_expired	|NAS Auth response timer expired	|AGW	|No	|MME	|
+|emm_status_rcvd	|EMM status received	|AGW	|No	|MME	|
+|emm_status_sent	|EMM status sent	|AGW	|No	|MME	|
+|nas_security_mode_command_timer_expired	|NAS security mode command timer expired	|AGW	|No	|MME	|
+|extended_service_request	|Extended service request	|AGW	|No	|MME	|
+|tracking_area_update_req	|Tracking area update request	|AGW	|No	|MME	|
+|tracking_area_update	|Tracking area update success	|AGW	|No	|MME	|
+|service_request	|Service request	|AGW	|Yes	|MME	|
+|security_mode_reject_received	|Security mode reject received	|AGW	|No	|MME	|
+|mme_new_association	|New SCTP association	|AGW	|No	|MME	|
+|ue_context_release_command_timer_expired	|UE context release command timer expired	|AGW	|No	|MME	|
+|enb_sctp_shutdown_ue_clean_up_timer_expired	|SCTP shutdown UE clean up timer expired	|AGW	|No	|MME	|
+|ue_context_release_req	|UE context release request	|AGW	|No	|MME	|
+|s1ap_error_ind_rcvd	|S1AP error indication received	|AGW	|No	|MME	|
+|s1_reset_from_enb	|S1 reset from eNB	|AGW	|No	|MME	|
+|nas_non_delivery_indication_received	|NAS non delivery indication received	|AGW	|No	|MME	|
+|spgw_create_session	|SPGW create session success	|AGW	|No	|MME	|
+|ue_pdn_connection	|UE PDN connection	|AGW	|No	|MME	|
+|ue_pdn_connectivity_req	|UE PDN connectivity request	|AGW	|No	|MME	|
+|	|	|	|	|	|
+|ue_reported_usage / up	|Reported TX traffic for subscriber / session in bytes	|AGW	|Yes	|sessiond	|
+|ue_reported_usage / down	|Reported RX traffic for subscriber / session in bytes	|AGW	|Yes	|sessiond	|
+|ue_dropped_usage / up	|Reported dropped TX traffic for subscriber / session in bytes	|AGW	|No	|sessiond	|
+|ue_dropped_usage / down	|Reported dropped RX traffic for subscriber / session in bytes	|AGW	|No	|sessiond	|
+
+More up to date information might be available from the “Metrics Explorer” in the metrics component.
+![alerts4](assets/nms/userguide/alerts/alerts4.png)
+**Custom Alert Configuration**
+
+An alert configuration consists of
+
+* Name/Description of the alert
+* Alert Definition
+* Alert receiver to be notified when the alert is fired
+* Additional labels which can be added to provide more information about the alert.
+
+Alert definition is the most interesting attribute in the configuration. It consists of a metric expression(based on prometheus promQL expression) and a duration attribute which specifies the time for the expression to be true, following which the alert is fired.
+![alerts5](assets/nms/userguide/alerts/alerts4.png)![alerts6](assets/nms/userguide/alerts/alerts4.png)
+
+Following examples shows how we can use the above mentioned metrics and add custom alerts based on them.
+
+**eNB Down Alert:**
+This alert fire if eNB Rf Tx is down in any of the gateways in your network.
+
+Expression:
+
+```
+sum by(gatewayID) (enodeb_rf_tx_enabled{networkID="<your network ID>"} < 1)
```

Comment:
```suggestion
enodeb_rf_tx_enabled{networkID="<your network ID>"} < 1
```
I'm not 100% sure about this metric but you don't need to sum by gatewayID since this metric should only have one series per gateway right?

Reply:
In HA feature, we can have something like this.